remote/src/main/java/com/kdjj/remote/datasource/RecipeListRemoteDataSourceImpl.kt

```diff
@@ -29,5 +38,20 @@ internal class RecipeListRemoteDataSourceImpl @Inject constructor(
     ): Result<List<Recipe>> =
         runCatching {
             recipeListService.fetchSearchRecipeListAfter(keyword, refresh)
+        }.errorMap {
+            fireStoreExceptionToDomain(it)
+        }
+
+    private fun fireStoreExceptionToDomain(throwable: Throwable) =
+        when (throwable) {
+            is FirebaseFirestoreException -> {
+                when (throwable.code.value()) {
+                    14 -> NetworkException()
+                    else -> ApiException()
+                }
+            }
+            else -> {
+                Exception(throwable)
+            }
```

Comment:
```suggestion
            else -> throwable
```
Exception을 다시 생성하지 않고 이렇게 되어도 상관 없을 것 같은데 어떻게 생각하시나요??

Reply:
errorMap 에서 어차피 Exception 으로 변환해야 하는데 여기서 변환하면 되지 않을까요?

Reply:
아하 그렇군요!